src/main/java/uk/gov/hmcts/reform/sscs/utility/StringUtils.java

```diff
@@ -45,4 +45,20 @@ public static String getInitalsAndSurnameFromName(String fullName) {
 
         return result.toString();
     }
+
+    public static String getMaskedEmail(String email){
+        String shortenedEmail = "";
+        if(email!=null && email.contains("@")){
+            shortenedEmail = org.apache.commons.lang3.StringUtils.left(email,3) + "***" + email.substring(email.indexOf("@"),email.indexOf("@")+3) + "***";
```

Comment:
can we add a static import of left() and right(), rather than including the package in the code